Modules/_ctypes/cfield.c

```diff
@@ -1198,13 +838,15 @@ c_set(void *ptr, PyObject *value, Py_ssize_t size)
 static PyObject *
 c_get(void *ptr, Py_ssize_t size)
 {
+    assert(NUM_BITS(size) || (size == sizeof(char)));
     return PyBytes_FromStringAndSize((char *)ptr, 1);
 }
 
 /* u - a single wchar_t character */
```

Comment:
To be consistent, how about changing `u: ` instead of `u -`? (look at other places as well if you want consistency)

Reply:
I don't really care for that level of consistency in the comments, but, why not :)